support/mock/mockrt.go

```diff
@@ -938,10 +973,6 @@ func (rt *Runtime) Verify() {
 		rt.failTest("missing expected batch verify seals with %v", rt.expectBatchVerifySeals)
 	}
```

Comment:
Insert failures if
- `expectAggregateVerifySeals` is not nil
- `gasCharged` is not empty